vesin/src/vesin.cpp

```diff
@@ -72,13 +66,9 @@ extern "C" int vesin_neighbors(
                 {{box[2][0], box[2][1], box[2][2]}},
             }}};
 
-            vesin::cpu::neighbors(
-                reinterpret_cast<const vesin::Vector*>(points),
-                n_points,
-                vesin::BoundingBox(matrix, periodic),
-                options,
-                *neighbors
-            );
+            vesin::cpu::neighbors(reinterpret_cast<const vesin::Vector*>(points), n_points, vesin::BoundingBox(matrix, periodic), options, *neighbors);
```

Comment:
same, this got reformated. Maybe because there is no line length limit in the clang-format config?